src/apps/bridge/memfault_platform_config.h

```diff
@@ -19,8 +19,3 @@
 #define MEMFAULT_METRICS_HEARTBEAT_INTERVAL_SECS (6 * 60 * 60)
 
 #define MEMFAULT_WATCHDOG_SW_TIMEOUT_SECS 30
-
-// If you want to enable heap stats collection while debugging, uncomment the following lines
-// #define MEMFAULT_COREDUMP_COLLECT_HEAP_STATS 1
-// #define MEMFAULT_FREERTOS_PORT_HEAP_STATS_ENABLE 1
-// #define MEMFAULT_COREDUMP_HEAP_STATS_LOCK_ENABLE 0
```

Comment:
Let's keep these commented lines in! It's tough to discover all these variables when this kind of debugging comes up. They're also a good reminder that this functionality is available.

Reply:
I think I accidently removed that file and then copied another directories `memfault_platform_config.h` file back into this directory.